### PR TITLE
[WIP] Config and sequence

### PR DESCRIPTION
**Pull request by felippenardi**
@raphamorim @emersonlaurentino 
This will allow us to do:
```js
withConfig(withTada(Component), { duration: '4s' })
```
and
```js
withSequence(withTada, withWobble, withShake, Component)
```

---

Can you guys give me early feedback letting me know your thoughts on the implementation?

### CODE REVIEW

File: react-motions/src/withConfig.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+function withConfig(Component, config) {
+  const { delay, duration, fillMode } = config;
+
+  if (Component.props.style) {
+    if (delay) {
+      Component.props.style['animationDelay'] = delay
+      Component.props.style['WebKitAnimationDelay'] = delay
```
Comment (emersonlaurentino) [inline react-motions/src/withConfig.js:9]: what do you think of the prop `duration` and `delay` being an integer?

and have something like that within the `withConfig`:

```jsx
Component.props.style['animationDelay'] = `${delay}s`
```

Comment (emersonlaurentino) [reply react-motions/src/withConfig.js:9]: Btw, the components props are type integer.

Comment (felippenardi) [reply react-motions/src/withConfig.js:9]: @emersonlaurentino That would limit setting animation for seconds. `ms` is also a valid value.

Comment (emersonlaurentino) [reply react-motions/src/withConfig.js:9]: It's true, we have to review this in the components.

Comment (felippenardi) [reply react-motions/src/withConfig.js:9]: @emersonlaurentino Maybe we can just assume that is the default and pass in fractioned numbers to represent milliseconds.

Comment (emersonlaurentino) [reply react-motions/src/withConfig.js:9]: i think better pass string with time in ms or s.

**Pull request closed without merging**